code/game/area/areas/ruins/lavaland.dm

```diff
@@ -32,9 +32,44 @@
 /area/ruin/powered/seedvault
 	icon_state = "dk_yellow"
 
-/area/ruin/powered/syndicate_lava_base
+/area/ruin/syndicate_lava_base
```

Comment:
/area/ruin/unpowered/syndicate_lava_base

Reply:
isn't unpowered permanently without power?

Reply:
nevermind, it's always_unpowered = FALSE i thought i read it saying TRUE.